openmrs-compose.yaml

```diff
@@ -37,7 +36,7 @@ services:
 
   openmrs:
     restart: "always"
-    image: openmrs/openmrs-reference-application-distro:nightly
+    image: ampathke/openmrs-analytics-distro:latest
```

Comment:
How easy is it for people outside AMPATH to update this image? If it is not easy, then I would prefer an `openmrs` image. Also maybe instead of `nightly` or `latest` we should use a specific tag, e.g., `2.10` such that if `latest` moves, it does not break our e2e tests.

Reply:
@bashir2 this is an openmrs image only that it has been hosted in the ampath org in docker hub
It's similar to ```openmrs-reference-application-distro:qa``` plus ```atom feed module```

Reply:
I see; yeah I think moving it under openmrs is better. BTW, where is the `Dockerfile` for this image? Should we add that to this code repository as well?